### ISSUE TITLE
any details regarding the experiments on the kvasir dataset?

### ISSUE THREAD

**Issue by uobinxiao** [tianrun-chen/SAM-Adapter-PyTorch#32]
Could you elaborate a little bit the evaluation protocol on the kavisr dataset? The data splition and others.

**Comment by tianrun-chen** [comment]
Hello! As we have mentioned in the main manuscript, we use the same protocol (including the test-train split) from Medico multimedia task at mediaeval 2020: Automatic polyp segmentation.

**Comment by uobinxiao** [comment]
thanks for your reply. I want to list your result as benchmark, but the test set of kavisr dataset doesn't contain ground truth. Did you send your results to the organizer to get the scores reported in your paper?
